src/main/kotlin/graphql/schema/idl/SchemaGeneratorHelperExt.kt

```diff
@@ -4,18 +4,18 @@ import graphql.introspection.Introspection
 import graphql.language.Directive
 import graphql.schema.GraphQLDirective
 import graphql.schema.GraphqlTypeComparatorRegistry
+import graphql.schema.idl.RuntimeWiring.MOCKED_WIRING
+import graphql.schema.idl.SchemaGenerator.Options.defaultOptions
 
 class SchemaGeneratorHelperExt : SchemaGeneratorHelper() {
     // Re-expose a package protected method as public
-    fun buildDirective(directive: Directive,
-                       graphQLDirective: GraphQLDirective,
-                       directiveLocation: Introspection.DirectiveLocation,
-                       comparatorRegistry: GraphqlTypeComparatorRegistry): GraphQLDirective {
-        // Note 1: for now, it seems safe to pass buildCtx = null, since the code path where buildCtx is used,
-        //         is never called (directive.name == graphQLDirective.name is always true, see line with
-        //         directiveDefOpt = FpKit.findOne ...)
-        // Note 2: repeatable directives (new feature in graphql-java 16) likely don't work,
+    fun buildAppliedDirective(directive: Directive,
+                              graphQLDirective: GraphQLDirective,
+                              directiveLocation: Introspection.DirectiveLocation,
+                              comparatorRegistry: GraphqlTypeComparatorRegistry): GraphQLDirective {
+        // Note: repeatable directives (new feature in graphql-java 16) likely don't work,
         //         since we don't pass in the full set of discovered directives
-        return super.buildDirective(null, directive, setOf(graphQLDirective), directiveLocation, comparatorRegistry)
+        val context = BuildContext(TypeDefinitionRegistry(), MOCKED_WIRING, emptyMap(), defaultOptions())
```

Comment:
Can't pass `null` here anymore.
This should keep working for now, but we might want to think of a better approach.